### PR TITLE
Convert pickle to jsonpickle for testdata in json

### PR DESCRIPTION
**Pull request by CoMPaTech**

### CODE REVIEW

**Comment by CoMPaTech** [comment]
Lets wait for feedback on https://github.com/home-assistant/core/pull/36371 before merging

**Review by bouwew — APPROVED** [review]
Looks good I think.

**Comment by CoMPaTech** [comment]
Seeing upstream commands, lets switch - testdata generation only so no version bump required

**Pull request merged**
